### DISCUSSION TITLE
FLC no longer usable to engage 'Selected Altitude' Mode

### DISCUSSION THREAD

**Discussion by alvoram** [flybywiresim/aircraft#4767] [Help] [answered]
Hi, hope somebody can help me. 

Recently changed to the dev model, trying out the new custom auto pilot.

On previous stable model, and original a320, FLC was bound to engage 'selected altitude' mode, and worked just like 'pulling' the real altitude knob on the MCP would.

I did this 2 ways, one, with my stream deck, I changed the altitude and hit FLC to engage selected altitude. 

But also, through the in game controller bindings, I have a button bound to 'Toggle FLC' on my throttle quadrant, which previously worked the same way, would engage 'selected altitude' mode, just like pulling the real altitude knob on the MCP. 

Now neither of these methods work. 

Is there a different binding I can use for my controller to engage 'selected altitude' mode?

And is there a command I can call with the 'flight tracker' stream deck plug in to do the same thing?

Thanks

Apologies in advance if this is difficult to understand, I am new to simming. :)

**Comment by tracernz** [comment, marked as answer]
https://github.com/flybywiresim/a32nx/blob/master/docs/a320-events.md

**Reply by alvoram** [reply]
thanks tracer, **IF** I understand this correct, they are 'sim connect events' that can be mapped to do exactly what I'm asking... Specifically A32NX.FCU_ALT_PULL does that job???

If so, it seems flight tracker doesn't support fbw's custom events yet... off to their github to do some persuading...

I guess there is no simple in game key binding that could do the same thing for now?

**Reply by tracernz** [reply]
Yes, that is correct. 👍
